kotlin-grpc/build.gradle.kts

```diff
@@ -0,0 +1,12 @@
+dependencies {
+    implementation(project(":kotlin"))
+    implementation(project(":grpc"))
+    implementation(project(":junit5"))
+
+    // kotlin
+    implementation(libs.kotlin.coroutines.jdk8)
+    implementation(libs.kotlin.reflect)
```

Comment:
Question: Is `libs.kotlin.reflect` used?

Reply:
Ref: https://github.com/line/armeria/pull/4724#discussion_r1127589412